api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -92,6 +98,37 @@ WorkspaceResponse toApiWorkspaceResponse(
       target = "researchOutcomeEnumSet",
       source = "researchOutcomeList",
       nullValuePropertyMappingStrategy = SET_TO_DEFAULT)
+
+  // Normally using ignore should be frowned upon. In a merge method
+  // like this one, it's unavoidable; otherwise we'd just make a straight-up translation.
+  // However,
+  @Mapping(target = "approved", ignore = true)
+  @Mapping(target = "billingAccountName", ignore = true)
+  @Mapping(target = "billingAccountType", ignore = true)
+  @Mapping(target = "billingMigrationStatusEnum", ignore = true)
+  @Mapping(target = "billingStatus", ignore = true)
+  @Mapping(target = "cdrVersion", ignore = true)
+  @Mapping(target = "cohorts", ignore = true)
+  @Mapping(target = "conceptSets", ignore = true)
+  @Mapping(target = "creationTime", ignore = true)
+  @Mapping(target = "creator", ignore = true)
+  @Mapping(target = "dataAccessLevel", ignore = true)
+  @Mapping(target = "dataAccessLevelEnum", ignore = true)
+  @Mapping(target = "dataSets", ignore = true)
+  @Mapping(target = "disseminateResearchSet", ignore = true)
+  @Mapping(target = "firecloudName", ignore = true)
+  @Mapping(target = "firecloudUuid", ignore = true)
+  @Mapping(target = "lastAccessedTime", ignore = true)
+  @Mapping(target = "lastModifiedTime", ignore = true)
+  @Mapping(target = "name", ignore = true)
+  @Mapping(target = "published", ignore = true)
+  @Mapping(target = "researchOutcomeSet", ignore = true)
+  @Mapping(target = "reviewRequested", ignore = true)
+  @Mapping(target = "timeRequested", ignore = true)
```

Comment:
question: By ignoring it here we are saying look at the corresponding method in IMPL to see how to convert it from DB to other modal (and vice cersa)?

Reply:
Ignore should be very rare. It's almost always a sign that we don't have a good handle on this. Basically, if a function returns an object, it should have all required fields populated. Swagger-codegen doesn't give us a good way to set some defaults.

I have a better solution in the works.